docs/components/concepts/access-control/authorizations.md

```diff
@@ -0,0 +1,150 @@
+
+## Default Roles
+
+Camunda provides predefined roles to simplify access management:
+
+| Role Name       | Purpose                                                                         | Typical Authorizations                                                                                                                              |
+| --------------- | ------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `admin`         | Full control over all orchestration resources and applications.                 | All permissions for all resources: `READ`, `CREATE`, `UPDATE`, `DELETE`, including `ACCESS` to all applications.                                    |
+| `readonlyadmin` | Audit-focused users who need read-only access across the orchestration cluster. | `READ` for all resources, including `READ_PROCESS_DEFINITION`, `READ_PROCESS_INSTANCE`, `READ_USER_TASK`, etc.                                      |
+| `connectors`    | Technical role for executing connector calls.                                   | `READ_PROCESS_DEFINITION` on **Process Definition** (`*`), `UPDATE_PROCESS_INSTANCE` on **Process Definition** (`*`), `CREATE` on **Message** (`*`) |
+| `rpa`           | Role for RPA workers.                                                           | `READ` on **Resource** (`*`), `UPDATE_PROCESS_INSTANCE` on **Process Definition** (`*`)                                                             |
+
+### Role Assignment in SaaS
+
+- **admin**: Automatically assigned to Organization Owner and Admin
+- **connectors**: Automatically assigned to Connectors Runtime in Cluster deployment
+- **readonlyadmin**: Automatically assigned to Camunda Support Agents for support cases
+
+## Common Use Cases
+
+### Application Access
+
+Users need specific application permissions to access Orchestration Cluster components::
+
+- **UI Access**: Resource Type `application` and Resource Key is one of the components Operate, Tasklist, Identity.
+  - Example: `operate` for Operate access
+  - Example: `tasklist` for Tasklist access
+- Without these permissions, users cannot log in to the compoennts
+
+### Resource Access
+
+Within applications, users need additional permissions for specific resources, e.g.:
+
+- **Process Related**
+  - `READ_PROCESS_DEFINITION` to view process models
+  - `CREATE_PROCESS_INSTANCE` to start new processes
+  - `UPDATE_PROCESS_INSTANCE` to modify running instances
+- **Decision Related**
+  - `READ_DECISION_DEFINITION` to view DMN models
+  - `CREATE_DECISION_INSTANCE` to execute decisions
+
+### API Access
+
+When implementing your own integrations by e.g. using a Camunda Client, you should consider the following:
+
+- **Job Workers**
+  - `UPDATE_PROCESS_INSTANCE` to complete jobs for the specific process definitions
```

Comment:
❓ Should we name the resource type here as well?

Reply:
added